app/models/workflow.rb

```diff
@@ -191,6 +202,17 @@ def important_note
     action if action.try(:request_type) == Action::IMPORTANT_NOTE
   end
 
+  # Temp-to-be-removed
+  # The method below are getters and setters for assigned_to that allows us to set the assigned_to_id and get assigned_to.
+  # We are unable to use belongs_to :assigned_to, class_name: "User as the User table is
```

Comment:
Missing a closing quote
```suggestion
  # We are unable to use belongs_to :assigned_to, class_name: "User" as the User table is
```